docs/reference/aggregations/bucket/frequent-items-aggregation.asciidoc

```diff
@@ -113,17 +119,20 @@ and (2.) from which cities they make those purchases. We are interested in sets
 with three or more items, and want to see the first three frequent item sets 
 with the highest support.
 
+Note that we use the <<async-search, async search>> endpoint in this first 
+example.
```

Comment:
doesn't this require changing this?

> The API returns a response similar to the following one:

Async returns an `id`, which than needs to be used to get the response.

Reply:
I've adjusted the docs so they include the missing step. However, I didn't add the response to the POST request as it would be long and the instructions are clear enough without it. Let me know if you think it would be better to add the response though.